docs/output.md

```diff
@@ -146,6 +147,19 @@ Optionally, the ITS region can be extracted from each ASV sequence using ITSx, a
 
 </details>
 
+#### assignSH
+
+Optionally, a UNITE species hypothesis (SH) can be added to the taxonomy. In short, the sequences are matched to the selected UNITE database with VSEARCH, using the sequence identity cutoff 98.5%. If a good enough and unique match is found, the sequence is assigned the SH of the matching sequence, and the taxonomy assignment for the sequence is changed to that of the SH. If no match is found, the taxonomy generated by assignTaxonomy and/or assignSpecies is kept.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `assignsh/`
+  - `ASV_tax_species_SH.tsv`: Taxonomic classification with SH taxonomy added in case of a match.
+  - `vsearch_blastout.tsv`: Raw vsearch results.
```

Comment:
Hm and what/where is the `*.txt` file from `VSEARCH_USEARCHGLOBAL`, i.e. https://github.com/jtangrot/ampliseq/blob/59a432805691fbca33a6b924f939fae03de6955a/conf/modules.config#L321-L330?

Reply:
Thanks, forgot to update that file...